src/modules/file/file.repository.ts

```diff
@@ -331,13 +347,17 @@ export class SequelizeFileRepository implements FileRepository {
   async getFilesWhoseFolderIdDoesNotExist(
     userId: File['userId'],
   ): Promise<number> {
+    const folders = await this.fileModel.findAll({
+      where: { userId },
+    });
+
+    const folderIds = folders.map((folder) => folder.folderId);
+
     const { count } = await this.fileModel.findAndCountAll({
       where: {
         folderId: {
           [Op.not]: null,
-          [Op.notIn]: this.fileModel.findAll({
-            where: { userId },
-          }),
+          [Op.notIn]: folderIds,
```

Comment:
This is probably the only change I made within a use case or repository, basically it seeks to avoid asynchrony errors in the query.